icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -3778,12 +3870,12 @@ void MeasureFormatTest::TestKilogramIdentifier() {
     assertEquals("nanogram", "", nanogram.getType());
     assertEquals("nanogram", "nanogram", nanogram.getIdentifier());
 
-    assertEquals("prefix of kilogram", UMEASURE_SI_PREFIX_KILO, kilogram.getSIPrefix(status));
-    assertEquals("prefix of gram", UMEASURE_SI_PREFIX_ONE, gram.getSIPrefix(status));
-    assertEquals("prefix of microgram", UMEASURE_SI_PREFIX_MICRO, microgram.getSIPrefix(status));
-    assertEquals("prefix of nanogram", UMEASURE_SI_PREFIX_NANO, nanogram.getSIPrefix(status));
+    assertTrue("prefix of kilogram", UMEASURE_SI_PREFIX_KILO == kilogram.getPrefix(status));
+    assertTrue("prefix of gram", UMeasurePrefix::ONE == gram.getPrefix(status));
```

Comment:
An example of the changes necessary if we choose to not include top-level symbols (via constexpr).